src/Proto3/Suite/DotProto/Parsing.hs

```diff
@@ -68,7 +69,17 @@ import qualified Turtle.Compat as Turtle (encodeString, fromText)
 parseProto :: Path -> String -> Either ParseError DotProto
 parseProto modulePath = parseProtoWithFile modulePath ""
 
-parseProtoWithFile :: Path -> String -> String -> Either ParseError DotProto
+-- | Parse a protobuf source string with an associated module path and file 
+-- path. 
```

Comment:
What are the module path and file path used for?  Just error messages?  Whatever the answer, I think it would be good to include that detail in the comment.